osdep/timer.c

```diff
@@ -49,6 +49,11 @@ int64_t mp_time_ns(void)
     return mp_raw_time_ns() - raw_time_offset;
 }
 
+int64_t mp_time_ns_convert(int64_t raw_time_ns)
```

Comment:
i am not sure if this is the best way to do this conversion. all timing info i get from the displaylink are in absolute raw mach units and need to be converted from mach > nano > mp time (subtracted offset).

Reply:
If it is really needed, what do you think about `mp_time_from_raw` and use it in existing code too?

``` diff
diff --git a/osdep/timer.c b/osdep/timer.c
index d0a8a92f8a..d9c008bf3e 100644
--- a/osdep/timer.c
+++ b/osdep/timer.c
@@ -44,9 +44,14 @@ void mp_time_init(void)
     mp_exec_once(&timer_init_once, do_timer_init);
 }

+int64_t mp_time_from_raw(uint64_t raw_time)
+{
+    return raw_time - raw_time_offset;
+}
+
 int64_t mp_time_ns(void)
 {
-    return mp_raw_time_ns() - raw_time_offset;
+    return mp_time_from_raw(mp_raw_time_ns());
 }

 double mp_time_sec(void)
```